compiler/luci/logex/src/CircleNodeSummaryBuilder.cpp

```diff
@@ -68,7 +68,7 @@ bool CircleNodeSummaryBuilder::build(const loco::Node *node, const locop::Symbol
     {
       if (i)
         ss << ",";
-      ss << node->dim(i).known() ? node->dim(i).value() : -1;
+      ss << (node->dim(i).known() ? node->dim(i).value() : -1);
```

Comment:
this is not shift op... 
what warning/error do we get from which compiler?

Reply:
Oh you're right. I'll recheck the warning. :cry:

Reply:
The full warning message as like following
```
warning: operator '?:' has lower precedence than '<<'; '<<' will be evaluated first [-Wparentheses]
```

The word 'shift' was mistakenly chosen, so I update the commit comment correctly.
Thanks.